static/js/tools/scatter/plot_options.tsx

```diff
@@ -84,36 +92,40 @@ function PlotOptions(): JSX.Element {
           <Col sm={1} className="plot-options-label">
             Log scale:
           </Col>
-          <Col sm="auto">
-            <FormGroup check>
-              <Label check>
-                <Input
-                  id="log-y"
-                  type="checkbox"
-                  checked={y.value.log}
-                  onChange={(e) => checkLog(y, e)}
-                />
-                {display.chartType === ScatterChartType.SCATTER
-                  ? "Y-axis"
-                  : y.value.statVarInfo.title || y.value.statVarDcid}
-              </Label>
-            </FormGroup>
-          </Col>
-          <Col sm="auto">
-            <FormGroup check>
-              <Label check>
-                <Input
-                  id="log-x"
-                  type="checkbox"
-                  checked={x.value.log}
-                  onChange={(e) => checkLog(x, e)}
-                />
-                {display.chartType === ScatterChartType.SCATTER
-                  ? "X-axis"
-                  : x.value.statVarInfo.title || x.value.statVarDcid}
-              </Label>
-            </FormGroup>
-          </Col>
+          {yMinMax[0] * yMinMax[1] > 0 && (
```

Comment:
nice! we should set a disabled state to the input instead - but this is good for now.

add a short comment though, about why this is disabled, for a future reader.

Reply:
done